charts/oes/charts/spinnaker/templates/configmap/halyard-init-script.yaml

```diff
@@ -44,16 +44,30 @@ data:
     # additionalInitScript
     {{ tpl .Values.halyard.additionalInitScript $ | indent 4 }}
     {{- end }}
-    {{- else }}
+    
+    {{- else if contains "git" .Values.gitopsHalyard.repotype }}
```

Comment:
this can be simply "eq" as repotype is a string not a list.
{{- else if eq .Values.gitopsHalyard.repotype "git" }}

Reply:
done